python/examples/14-broadcast/main.py

```diff
@@ -0,0 +1,58 @@
+from uagents import Agent, Bureau, Context, Model, Protocol
+from uagents.setup import fund_agent_if_low
+
+# create agents
+# alice and bob will support the protocol
+# charles will try to reach all agents supporting the protocol
+alice = Agent(name="alice", seed="alice recovery phrase")
+bob = Agent(name="bob", seed="bob recovery phrase")
+charles = Agent(name="charles", seed="charles recovery phrase")
+
+fund_agent_if_low(alice.wallet.address())
+fund_agent_if_low(bob.wallet.address())
+fund_agent_if_low(charles.wallet.address())
+
+
+class Request(Model):
+    pass
+
+
+class Response(Model):
+    text: str
+
+
+# define protocol
+proto = Protocol(name="proto", version="1.0")
```

Comment:
Suggest giving these all more descriptive names like `BroadcastExampleRequest` to reduce the chance that this example will send messages to other agents with a generic Request/Response protocol.

Reply:
You are absolutely right. Will change those names.